src/components/sbb-toggle/sbb-toggle.tsx

```diff
@@ -188,12 +188,7 @@ export class SbbToggle implements ComponentInterface {
   public render(): JSX.Element {
     return (
       <Host role="radiogroup">
-        <div
-          class="sbb-toggle"
-          // tabIndex={this.disabled ? -1 : 0}
-          tabindex="-1"
-          ref={(toggle) => (this._toggleElement = toggle)}
-        >
+        <div class="sbb-toggle" tabindex="-1" ref={(toggle) => (this._toggleElement = toggle)}>
```

Comment:
Do you really need the reference _toggleELement as e.g. in connectedCallback you use directly "this._element.firstElementChild"?. Or would a getter be an option?

Reply:
They're actually two different elements, `this._element.firstElementChild` should return the first toggle option, while the `this._toggleElement` is just the toggle element of course. I observe the `firstElementChild` (i.e. the option) in the connectedCallback in order to have more stability when setting the checked pill position (to solve that visual bug we noticed last time if you remember).

Reply:
I don't get how firstElementChild can be the first slotted option :D. I have to to get deeper later.

Reply:
@jeripeierSBB Do you mean `this._element.shadowRoot.firstElementChild`?

Reply:
Yes, we marco pointed me to it as well in teams :-)